CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Don't generate fields that represent compositions of aspects within mutation types that represent services
 - Moved the `GraphQLAdapter` module to the root directory (`index.js`), simplifying the import process and reducing the required typing.
+- Disabled conjunction for the following operators:
+  - `eq` (Equal)
+  - `gt` (Greater Than)
+  - `ge` (Greater Than or Equal)
+  - `le` (Less Than or Equal)
+  - `lt` (Less Than)
+  - `startswith`
+  - `endswith`
```

Comment:
> on the same field

👍

As for the `+` operator, adding it would actually violate the [MD004/ul-style](https://github.com/DavidAnson/markdownlint/blob/v0.28.2/doc/md004.md) rule.

Reply:
Did not know about this. But doesn't the last example explain that it is ok as long as the symbols are consistent for each indentation level?

I think we should use `+` for indentations as that is what we have consistently done in changelogs of other `cds` packages.

Reply:
I just received a warning in VSCode. So, stick with the `+` operator.

<img width="1129" alt="MD004:ul-style" src="https://github.com/cap-js/graphql/assets/1238862/1c4bf471-5880-40aa-b297-151c776c074e">